pkg/astimport/astimport.go

```diff
@@ -139,7 +139,7 @@ func (i *Importer) ImportVariableDefinition(ref int, from, to *ast.Document) int
 
 func (i *Importer) ImportVariableDefinitions(refs []int, from, to *ast.Document) []int {
 	definitions := make([]int, len(refs))
-	for j, k := range definitions {
+	for j, k := range refs {
```

Comment:
Ouch, thanks for finding this hidden bug

Maybe also change it to `for i` ?

Reply:
the importer receiver is already `i`. Or do you mean something different?

Reply:
I mean `for i := 0` instead of range